### ISSUE TITLE
Easy copy of a whole chapter

### ISSUE THREAD

**Issue by ReneSci** [andrejv/wxmaxima#860]
If the user want to copy a whole chapter in a wxMaxima work-sheet and therefore selects all cells and sub-chapters - then, if one of the sub-chapters happens to be "closed"/compressed (what do you call it?), all the cells herein won't be copied - can this be changed ?

Or a better idea: a whole chapter with its sub-chapters and all sub-cells can be copied by just selecting the header in some way. Something like the feature you have made where the user can evaluate a whole chapter/sub-chapter by its header.

**Comment by gunterkoenigsmann** [comment]
This turned out to be easier than I thought.
Anyway, I would be grateful for any testing of the new functionality as (in order to simplify the code) I want to re-use the copy-and-paste functionality for saving .wxm files in a later step. 

Thanks a lot, and kind regards,
Gunter.

**Comment by ReneSci** [comment]
Sounds great !
Sorry for the delay.

I tried to download the nightly build of Maxima/wxMaxima 27 Marts - version.
I tried both the 64 and the 32-bit version.

wxMaxima can not load the attached file correctly. Especially all the
headings are missing - only the heading/subheading numbers are appering. -
Any clue ?

The mightly build on the 16 Marts had the same fault.
Kind regards
Rene'


2017-03-19 10:15 GMT+01:00 Gunter Königsman <notifications@github.com>:

**Comment by ReneSci** [comment]
Forgot to tell: the files are created using wxMaxima version 16.4.2
/Rene'




---------- Forwarded message ----------
From: Rene Hansen <reneshansen@gmail.com>
Date: 2017-03-27 12:56 GMT+02:00
Subject: Re: [andrejv/wxmaxima] Easy copy of a whole chapter (#860)
To: andrejv/wxmaxima <
reply@reply.github.com
>


Sounds great !
Sorry for the delay.

I tried to download the nightly build of Maxima/wxMaxima 27 Marts - version.
I tried both the 64 and the 32-bit version.

wxMaxima can not load the attached file correctly. Especially all the
headings are missing - only the heading/subheading numbers are appering. -
Any clue ?

The mightly build on the 16 Marts had the same fault.
Kind regards
Rene'


2017-03-19 10:15 GMT+01:00 Gunter Königsman <notifications@github.com>:

**Comment by gunterkoenigsmann** [comment]
Seems like github has removed the attachment. If you rename the file to something ending in ".zip"  or send it to me per Mail (I am Gunter and my homepage is physikbuch.de) I will try to find out what went wrong here.

**Comment by ReneSci** [comment]
Semms like,

If I, after undinstalling the previous version, deletes the registry key
(and all sub-keys) "HKEY_CURRENT_USER\Software\wxMaxima" and then installs
the nightly build - it seems to work.
Kind regards
Rene'


2017-03-28 12:18 GMT+02:00 Gunter Königsman <notifications@github.com>:

**Comment by gunterkoenigsmann** [comment]
Andrej has found and resolved the problem in the meantime: Sometimes if a selected font wasn't found there was no fallback.

**Comment by ReneSci** [comment]
Sounds good

By the way - I have tested the nice new function "Easy copy of a whole
chapter". It works quite fine. It only works on a collapsed
chapter/section, I suppose it is by intension.

I found one small bug:
If I collapse a whole chapter with some sub-sections herein, and if some of
these sub-sections are collapsed and others not, the headers of the
un-collapsed sections are not copied.
Kind rehards
Rene'


2017-05-17 17:09 GMT+02:00 Gunter Königsman <notifications@github.com>:
